api/seeds/shared/states.js

```diff
@@ -1,5 +1,17 @@
+const chalk = require('chalk');
+const logger = require('../../logger')('state seeder');
 const { states } = require('../../util/states');
 
 exports.seed = async knex => {
-  await knex('states').insert(states);
+  logger.info(`Begging to seed the ${chalk.cyan('states')} table`);
+  const total = await knex('states')
+    .count('id')
+    .first();
+  logger.info(`${chalk.cyan('states')} table currently has ${total.count}`);
+  if (total.count.toString() === '0') {
```

Comment:
Is this right?

Reply:
The documentation says it comes back as a bigint or a string, so I converted it to a string just in case and tested it that way.

Reply:
But also when I tested it on my system it did wipe out the states and saw that it was empty and added them back without duplicating them.